### DISCUSSION TITLE
Revert a single version in a PR

### DISCUSSION THREAD

**Discussion by mikethecalamity** [renovatebot/renovate#31563] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab

### Please tell us more about your question or problem

I've seen on Github projects using dependabot can comment on the PR to have it undo the version upgrade for a specific dependency. Is something like this possible with Renovate?

**Comment by RahulGautamSingh** [comment, marked as answer]
There is no such functionality.
